### ISSUE TITLE
highlights/freehand on a bookmarked pages gets copied to other bookmarked pages. 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#694]
```
What steps will reproduce the problem?
1.highlight or freehand something on a page
2. Bookmark that page
3. Bookmark another random page 

What is the expected output?

What do you see instead?

The highlights /freehand from  first bookmarked page appears on the second 
bookmarked page too.

What version of the product are you using?
Latest 

On what operating system?
Kitkat 

What device are you using?


Please provide any additional information below.

```

Original issue reported on code.google.com by `PARTHDES...@gmail.com` on 12 Jan 2014 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
Pls provide step-by-step screenshots.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 12 Jan 2014 at 9:51
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
1)I drew some "free hand" on the page 117(first image) then I bookmarked this 
page. 
2) Later I bookmarked the page number 121,and the "free hand" From page 117 got 
copied here by itself.(second image)
```

Original comment by `PARTHDES...@gmail.com` on 12 Jan 2014 at 10:01

Attachments:
- [Screenshot_2014-01-12-15-24-59.png](https://storage.googleapis.com/google-code-attachments/ebookdroid/issue-694/comment-2/Screenshot_2014-01-12-15-24-59.png)
- [Screenshot_2014-01-12-15-25-30.png](https://storage.googleapis.com/google-code-attachments/ebookdroid/issue-694/comment-2/Screenshot_2014-01-12-15-25-30.png)

**Comment by GoogleCodeExporter** [comment]
```
So on further observation I figured that the problem may not have much to do 
with the bookmarking... The highlight/freehand from one page randomly gets 
copied to some other page. 
```

Original comment by `PARTHDES...@gmail.com` on 12 Jan 2014 at 4:51

**Comment by GoogleCodeExporter** [comment]
```
Thanks for report. Fixed in 2.1.2

```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 13 Jan 2014 at 3:01
- Changed state: **Fixed**
